docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>Since the data is hosted on Google drive, we've <a href="https://github.com/TDAmeritrade/stumpy/issues/270" target="_blank">started backing up the relevant data sets on Zenodo</a>. Would you mind doing the same with the data sets used in this tutorial? </p><p> </p><p>In the future, we plan to update/migrate all of our tutorials to download their data from Zenodo directly so this could be greatly simplified.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>So if I upload the data to Zenodo I can also get rid of the functions for retrieval from Google Drive?</p>

Reply:
Yes, I think that would be appropriate!

Reply:
<p>I uploaded the dataset to Zenodo. Do you need to approve it?</p><p> </p>

Reply:
<p>Ah, scratch that. I can simply download it from there. I feel like when I used internet forums for the very first time in my life, haha.</p>

Reply:
Yes, approval is on me. However, I noticed that you've uploaded the entire zip file. Instead, we only want the final CSV to be uploaded and the retrieval of files in the tutorials to look like:

```
df = pd.read_csv("zenodo_url")
```

Currently, other CSVs have been uploaded to Zenodo [following this approach](https://github.com/TDAmeritrade/stumpy/blob/master/docs/Zenodo.ipynb). From what I see, you'll have to upload each `eye[i]` data, the `sentence` data, and each organism for `data` in the mtDNA.  

If this is too much work then leave it for now I can handle it afterward. I am going to need to "reject" (nothing personal) the existing uploads as we are only looking for the CSV.